packages/zipkin-context-cls/test/CLSContext.test.js

```diff
@@ -84,6 +85,59 @@ function CLSContextPerAsync(supportAsync) {
       setTimeout(callback, 10);
     });
   });
+
+  it('supports callbacks on eventEmitter', async() => {
+    const eventEmitter = new EventEmitter();
+    const ctx = new CLSContext(namespace, supportAsync);
+    // if you comment this out the tests will fail
+    // Apparently, it is necessary to explicitly register event emitters with context
+    ctx.bindEmitter(eventEmitter);
+    // const dbg = (...args) => console.log(...args);
```

Comment:
I actually think we need loggers in this kind of tests. What if we make the logger enabled or disabled based on `logLevel`? see https://karma-runner.github.io/4.0/config/configuration-file.html